sdk/Lusid.Sdk.Tests/Utilities/InstrumentExamples.cs

```diff
@@ -155,6 +155,93 @@ internal static LusidInstrument CreateExampleSwap()
             );
         }
 
+        internal static InterestRateSwaption CreateExampleInterestRateSwaption()
+        {
+                        // CREATE an Interest Rate Swap (IRS)
+            var startDate = new DateTimeOffset(2020, 2, 7, 0, 0, 0, TimeSpan.Zero);
+            var maturityDate = new DateTimeOffset(2030, 2, 7, 0, 0, 0, TimeSpan.Zero);
+
+            // CREATE the flow conventions, index convention for swap
+            var flowConventions = new FlowConventions(
+                scope: null,
+                code: null,
+                currency: "GBP",
+                paymentFrequency: "6M",
+                rollConvention: "MF",
+                dayCountConvention: "Act365",
+                paymentCalendars: new List<string>(),
```

Comment:
i presume the calendars are optional? they should be defaulted

Reply:
Currently, this field is non-optional so must be specified. But you raise a good point, we probably should allow it to be optional (within Lusid.All). Can discuss with Paul.